docs/architecture/architecture_overview.md

```diff
@@ -1,133 +1,172 @@
+* Orange lines indicate manual data flows, such as an analyst executing a Jupyter notebook
+* Yellow nodes indicate testing/development environments
+```{mermaid}
+flowchart LR
+%% default styles
+classDef default fill:white, color:black, stroke:black, stroke-width:1px
+linkStyle default stroke:black, stroke-width:4px
+classDef test fill:#fdfcd8, color:#000000
+classDef group fill:#cde6ef, color:black, stroke-width:0px
+classDef subgroup fill:#14A6E0, color:white
 
-The "production" ("prod") environment consists of:
-* RT Archiver
-* Airflow as run through Composer
-* `cal-itp-data-infra` project in Google Cloud Platform (BigQuery and Google Cloud Storage)
-    * Specifically, the `gtfs-data` and `littlepay-data-extract-prod` buckets in Google Cloud Storage
+%% note that you seemingly cannot have a subgraph that only contains other subgraphs
+%% so I am using "label" nodes to make sure each subgraph has at least one direct child
+subgraph sources[ ]
+    data_sources_label[Data Sources]:::group
+    raw_gtfs[Raw GTFS schedule data]
+    airtable[<a href='https://airtable.com/'>Airtable</a>]
+    raw_payment[Raw fare payment]
+    raw_rt[Raw GTFS RT feeds]
+end
+subgraph rt_archiver[ ]
+    rt_archiver_label[<a href='https://github.com/cal-itp/data-infra/tree/main/services/gtfs-rt-archiver-v3'>RT archiver</a>]:::group
+    prod_rt_archiver[Prod archiver]
+    test_rt_archiver[Test archiver]:::test
+end
+subgraph airflow[ ]
+    airflow_label[Airflow]:::group
+    airflow_prod[Production Airflow <br><i><a href='https://console.cloud.google.com/composer/environments?project=cal-itp-data-infra&supportedpurview=project'>Composer</a></i>]
+    airflow_local[Local Airflow <br><i><a href='https://github.com/cal-itp/data-infra/blob/main/airflow/README.md'>Setup</a></i>]:::test
+end
+subgraph gcp[Google Cloud Project]
+    subgraph bigquery[<a href=''>BigQuery</a>]
+        bq_cal_itp_data_infra[(cal-itp-data-infra)]
+        bq_cal_itp_data_infra_staging[(cal-itp-data-infra-staging)]:::test
+    end
+    subgraph gcs[<a href='https://console.cloud.google.com/storage/browser'>Google Cloud Storage</a>]
+        gcs_raw[(Raw)]
+        gcs_parsed[(Parsed)]
+        gcs_validation[(Validation)]
+        gcs_analysis[(Analysis artifacts)]
+        gcs_map_tiles[(Map tiles/GeoJSON)]
+        gcs_other[(Backups, Composer code, etc.)]
+        gcs_test[(test-* buckets)]:::test
+    end
+end
+
+subgraph consumers[ ]
+    consumers_label[Data consumers]:::group
+    jupyterhub[<a href='https://hubtest.k8s.calitp.jarv.us/hub/'>JupyterHub</a>]
+    metabase[<a href='https://dashboards.calitp.org/'>Metabase - dashboards.calitp.org</a>]
+    reports_website[<a href='https://reports.calitp.org'>reports.calitp.org</a>]
+end
+
+%% subgraphs cannot be styled in-line
+class sources,rt_archiver,airflow,gcp,consumers group
 
+%% manual data consumption; put first for easier style indexing
+%% add indices to linkStyle as new manual connections exist
+gcs_parsed --> jupyterhub
```

Comment:
I actually think let's maybe remove this line? I guess it's technically true that JupyterHub can read from parsed data but I think it's not a core enough workflow to be worth highlighting. I also think that removing this line might help the overall formatting become a lot clearer.